### PR TITLE
Acrolinx quality improvements: hcn-declaration-handles

### PR DESCRIPTION
**Pull request by gswashington**

### CODE REVIEW

**Comment by Jak-MS** [comment]
#assign: @robinharwood, @Xelu86 

Can you review the proposed changes? 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/networking/technologies/hcn/hcn-declaration-handles.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 
 # RPC context handles for HCN
 
->Applies to: Windows Server 2022, Windows Server 2019
+Applies to: Windows Server 2022, Windows Server 2019
 
 ## HCN_Network
 
-An HCN Network is an entity which is used to represent a host compute network and its associated system resources and policies. For example, a HCN network will typically consist of a set of metadata (e.g. id, name, type), a virtual switch, a host virtual network adapter (which acts as a default gateway for the network), a NAT instance (if required by the network type), a set of subnet and MAC pools, and any network-wide policies to be applied (e.g. ACLs).
+An HCN Network is an entity that is used to represent a host compute network and its associated system resources and policies. For example, a HCN network will typically consist of a set of metadata (for example, id, name, type), a virtual switch, a host virtual network adapter (that acts as a default gateway for the network), a NAT instance (if required by the network type), a set of subnet and MAC pools, and any network-wide policies to be applied (for example, ACLs).
```
Comment (robinharwood) [inline WindowsServerDocs/networking/technologies/hcn/hcn-declaration-handles.md:17]: The sentence beginning for example, its quite complex. Are you able to split this up or simplify the sentence?

File: WindowsServerDocs/networking/technologies/hcn/hcn-declaration-handles.md [hunk 2]
```diff
@@ -143,19 +144,19 @@ HcnCloseNetwork(
 
 ## HCN_Endpoint
 
-A HCN Endpoint is an entity which is used to represent an IP endpoint on a HCN network and its associated system resources and policies. For example, a HCN endpoint will typically consist of a set of metadata (e.g. id, name, parent network id), its network identity (e.g., IP address, MAC address), and any endpoint specific policies to be applied (e.g. ACLs, routes).
+A HCN Endpoint is an entity which is used to represent an IP endpoint on an HCN network and its associated system resources and policies. For example, an HCN endpoint will typically consist of a set of metadata (for example, id, name, parent network id), its network identity (for example, IP address, MAC address), and any endpoint specific policies to be applied (for example, ACLs, routes).
```
Comment (robinharwood) [inline WindowsServerDocs/networking/technologies/hcn/hcn-declaration-handles.md:147]:
```suggestion
A HCN Endpoint is an entity, which is used to represent an IP endpoint on an HCN network and its associated system resources and policies. For example, an HCN endpoint will typically consist of a set of metadata (for example, id, name, parent network id), its network identity (for example, IP address, MAC address), and any endpoint specific policies to be applied (for example, ACLs, routes).
```

Comment (robinharwood) [inline WindowsServerDocs/networking/technologies/hcn/hcn-declaration-handles.md:147]: The sentence beginning for example, its quite complex. Are you able to split this up or simplify the sentence?

**Review by robinharwood — APPROVED** [review]

**Comment by robinharwood** [comment]
Looks good, merging now.

**Pull request merged**